### DISCUSSION TITLE
When is the group tabs feature coming?

### DISCUSSION THREAD

**Discussion by Dean-Corso** [zen-browser/desktop#1809] [Q&A]
Hey guys,

just would like to ask when this feature is coming in ZEN? On the main page I can see its coming soon but when exactly round about? How will it work? Similar like in Chrome?
![T_2024-09-30_014011](https://github.com/user-attachments/assets/6c28dc69-a028-430c-bd7b-275d1a08f182)

**Comment by cyrneko** [comment]
Generally, no ETAs. Features and updates come when they are ready.
